README.md

```diff
@@ -117,7 +117,7 @@ Change the main branch to the right one, execute the following command:
 Example commands:
 
 ```sh
-  cd my-folder
+  cd my-flder
   git clone git@github.com:myaccount/my-project.git
 ```
```

Comment:
- [ ] Similarly, in your queries.sql file, all codes after line 121 are not necessary for this milestone and should not be included in this milestone. Please remove all of it.